x-pack/plugins/actions/server/builtin_action_types/swimlane/index.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { curry } from 'lodash';
+import { i18n } from '@kbn/i18n';
+import { schema } from '@kbn/config-schema';
+import { Logger } from '@kbn/logging';
+import { ActionType, ActionTypeExecutorOptions, ActionTypeExecutorResult } from '../../types';
+import { ActionsConfigurationUtilities } from '../../actions_config';
+import {
+  SwimlaneExecutorResultData,
+  SwimlanePublicConfigurationType,
+  SwimlaneSecretConfigurationType,
+  ExecutorParams,
+  ExecutorSubActionCreateRecordParams,
+} from './types';
+import { validate } from './validators';
+import {
+  ExecutorParamsSchema,
+  SwimlaneSecretsConfiguration,
+  SwimlaneServiceConfiguration,
+} from './schema';
+import { createExternalService } from './service';
+import { api } from './api';
+
+interface GetActionTypeParams {
+  logger: Logger;
+  configurationUtilities: ActionsConfigurationUtilities;
+}
+
+const supportedSubActions: string[] = ['application', 'createRecord'];
+
+// action type definition
+export function getActionType(
+  params: GetActionTypeParams
+): ActionType<
+  SwimlanePublicConfigurationType,
+  SwimlaneSecretConfigurationType,
+  ExecutorParams,
+  SwimlaneExecutorResultData | {}
+> {
+  const { logger, configurationUtilities } = params;
+
+  return {
+    id: '.swimlane',
+    minimumLicenseRequired: 'gold',
```

Comment:
I am not sure about that. Is it decided between the PM people of Elastic and Swimlane?

Reply:
@shimonmodi Can you confirm?